src/test/java/com/google/cloud/tools/appengine/cloudsdk/PathResolverTest.java

```diff
@@ -80,7 +81,12 @@ public void testGetLocationFromLink_valid() throws IOException {
     PathResolver.getLocationsFromLink(possiblePaths, gcloud);
 
     Assert.assertEquals(1, possiblePaths.size());
-    Assert.assertEquals(gcloud.getParent().getParent().toString(), possiblePaths.get(0));
+   if (SystemUtils.IS_OS_MAC_OSX) {
+     Assert.assertEquals("/private" + gcloud.getParent().getParent().toString(),  possiblePaths.get(0));
```

Comment:
I think did something dumb when I wrote this test, but I believe we can just do

`sdkHome.toRealPath().toString()` and it should work in both environments.